crates/runtime/src/env/memory/template_store.rs

```diff
@@ -52,7 +50,7 @@ where
 
         hash.and_then(|h| {
             self.bytes
-                .get(&h)
+                .get(h as &[u8])
```

Comment:
I've never encountered that. It uses `Deref` here?

Reply:
No, it doesn't use any trait. Converting arrays to slices with `as` is a primitive feature. What I didn't realize, however, is that we don't need this at all. `.get(h)` works fine now that `TemplateHash` is a `type` and not a `struct`.